.github/workflows/prod-build.yml

```
@@ -212,10 +218,11 @@ jobs:
           BLOG_ROOT: ${{ github.workspace }}/mdn/mdn-studio/content/posts
           CURRICULUM_ROOT: ${{ github.workspace }}/mdn/curriculum
           GENERIC_CONTENT_ROOT: ${{ github.workspace }}/mdn/generic-content/files
+          YARI_ROOT: ${{ github.workspace }}/mdn/yari
           BASE_URL: "https://developer.mozilla.org"
 
           # rari
-          BUILD_OUT_ROOT: "client/build"
+          BUILD_OUT_ROOT: "out"
```

Nit: This could be the job-level env:

```suggestion
          BUILD_OUT_ROOT: ${{ github.workspace }}/mdn/fred/out
```